### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cmtso**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +15,27 @@
...
+## Installation
+
+First, clone this repository:
+
+```
+# Clone this repository
+git clone https://github.com/<username>/<PROJECT-NAME>.git
+cd <PROJECT-NAME>
+```
+Alternatively, download the code as a .zip folder by going clicking the blue "Code" button on GitHub.
```
Comment (dc2917) [inline README.md]: Not sure about anyone else but my "Code" button is green

Comment (dc2917) [inline README.md]:
```suggestion
Alternatively, download the code as a .zip folder by clicking the "Code" button on GitHub, then clicking "Download ZIP".
```

File: README.md [hunk 2]
```diff
@@ -13,3 +15,27 @@
   - [ ] `<PROJECT-DOI>`
   - [ ] `<GITHUB-ORGANISATION>`
   - [ ] `<GITHUB-ORGANISATION>`
+- Change links or mention of username or  `PROJECT-NAME` in this file.
```
Comment (dc2917) [inline README.md]:
```suggestion
- Change links or mention of username or `PROJECT-NAME` in this file.
```

**Review by dc2917 — CHANGES_REQUESTED** [review]
I've made a few minor suggestions but otherwise I think this is a good start!

**Review by davidwyld — APPROVED** [review]

**Review by dc2917 — APPROVED** [review]
LGTM

**Pull request merged**